labelanalysis/serializers.py

```diff
@@ -22,21 +23,25 @@ def to_internal_value(self, commit_sha):
             commitid=commit_sha,
         ).first()
         if commit is None:
+            metrics.incr("label_analysis_request.errors.commit_not_found")
             raise exceptions.NotFound(f"Commit {commit_sha[:7]} not found.")
         if commit.staticanalysissuite_set.exists():
             return commit
         if not self.accepts_fallback:
+            metrics.incr("label_analysis_request.errors.static_analysis_not_found")
             raise serializers.ValidationError("No static analysis found")
         attempted_commits = []
         for _ in range(10):
             attempted_commits.append(commit.commitid)
             commit = commit.parent_commit
             if commit is None:
+                metrics.incr("label_analysis_request.errors.static_analysis_not_found")
```

Comment:
should we differentiate this from the previous metric?

Reply:
I don't think so... it's the same error, but in the 2nd case we try to fallback.
But they track the same thing.

Same reasoning for the metric below this one.